main.py

```diff
@@ -1,12 +1,185 @@
 import requests
+from customer import Customer
+from video import Video
 
-URL = "http://127.0.0.1:5000"
-BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
+# URL = "http://127.0.0.1:5000"
+# BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
-def main():
-    print("WELCOME TO RETRO VIDEO STORE")
-    pass
 
+# def main():
+#     print("WELCOME TO RETRO VIDEO STORE")
+#     pass
 
-if __name__ == "__main__":
-    main()
\ No newline at end of file
+
+def print_stars():
+    print("\n**************************\n")
+
+
+def list_options():
+
+
+    options = {
+        "1": "Create a customer", 
+        "2": "Update a customer",
+        "3": "Delete a customer",
+        "4": "List all customers", 
+        "5": "List details for one customer",
+        "6": "Create a video", 
+        "7": "Update a video",
+        "8": "Delete a video",
+        "9": "List all videos",
+        "10": "List details for one video"
+        }
+
+    print_stars()
+
+    print("Welcome to the Video Store CLI")
+    print("These are the actions you can perform")
+
+    print_stars()
+    
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    print_stars()
+
+    return options
+
+
+def make_choice(options):
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("What would you like to do?")
+        choice = input("Make your selection using the option number: ")
+
+    # if choice in ['4','5','6','7'] and task_list.selected_task == None:
+    #     print("You must select a customer before updating it, deleting it, marking it complete, or marking it incomplete.")
+    #     print("Let's select a customer!")
+    #     choice = "3"
+    
+    return choice
+
+
+
+def run_cli(play=True):
+
+    #initialize 
+    customer = Customer(url="https://localhost:5000/")
+    video = Video(url="https://localhost:5000/")
+
+    while play==True:
+
+        # print choices
+        options = list_options()
+        choice = make_choice(options)
+
+        # create a customer
+        if choice == "1":
+            print("Great! Let's create a new customer.")
+            name = input("What is the customer's name? ")
+            postal_code = input("What is the customer's postal code? ")
+            phone_number = input("What is the customer's phone number? ")
+
+            response = customer.create_customer(customer_name=name, postal_code=postal_code, phone_number=phone_number)
+            print_stars()
+            print("New task:", response["customer"])
```

Comment:
This assumes success. The responses from the API include status codes, which can be used to evaluate whether or not the API call succeeded. Success or fail messages can be displayed based on those status codes.